### PR TITLE
Detail View UX

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/entity-browser/src/components/DetailView/FieldErrorList.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+const FieldErrorList = props => {
+  if (!props.errors) {
+    return null
+  }
+
+  let errorValues = []
+  Object.keys(props.errors).map(key => {
+    errorValues = [...errorValues, ...props.errors[key]]
+  })
```
Comment (rzueger) [inline packages/entity-browser/src/components/DetailView/FieldErrorList.js]: Maybe it doesn't make sense to create a new `errorValues` array and throw the old one away for each errors entry. Maybe this results in a garbage collection overhead.

File: packages/entity-browser/src/util/detailView/reduxForms.spec.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
+import * as reduxForms from './reduxForms'
+
+describe('entity-browser', () => {
+  describe('util', () => {
+    describe('detailView', () => {
+      describe('reduxForm', () => {
```
Comment (rzueger) [inline packages/entity-browser/src/util/detailView/reduxForms.spec.js]: 'reduxForms'?

File: packages/entity-browser/src/util/detailView/reduxForms.spec.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+              lastname: 'griffin',
+              bool: false,
+              array1: [1, 2, 3],
+              array2: [1, 2, 4]
+            }
+            const diryFields = reduxForms.getDirtyFields(initialValues, values)
+
+            expect(diryFields).to.eql(['firstname', 'bool', 'array2'])
+          })
+        })
```
Comment (rzueger) [inline packages/entity-browser/src/util/detailView/reduxForms.spec.js]: (insert blank line?)

File: packages/entity-browser/src/util/detailView/reduxForms.spec.js (hunk 2 above)
Comment (MethodenMann) [inline packages/entity-browser/src/util/detailView/reduxForms.spec.js]: i renamed the file

File: packages/entity-browser/src/components/DetailView/DetailForm.js [hunk 4]
```diff
@@ -1,108 +1,135 @@
 import React from 'react'
+import {reduxForm, Field} from 'redux-form'
 import _get from 'lodash/get'
-import LabeledField from './LabeledField'
-import {Field, reduxForm} from 'redux-form'
+
```
Comment (rzueger) [inline packages/entity-browser/src/components/DetailView/DetailForm.js]: (two blank lines?)

**Pull request merged**